src/wms/src/test/java/org/geoserver/wms/wms_1_1_1/GetFeatureInfoTest.java

```diff
@@ -872,4 +876,46 @@ public void testScaleTolerance() throws Exception {
        assertPixel(image, 150, 150, Color.BLUE);
    }
    
+   /**
+    * Test GetFeatureInfo on a group layer with some no-queryable layers
+    * @throws Exception
+    */
+   @Test
+   public void testGroupLayerWithNotQueryableLayers() throws Exception {
+       Catalog catalog = getCatalog();
+       CatalogFactory factory = catalog.getFactory();
+       WorkspaceInfo workspace = catalog.getWorkspaceByName(MockData.CITE_PREFIX);
+       String groupLayer = "glqueryable";
+       
+       LayerInfo buildingsLayer = ModificationProxy.unwrap(catalog.getLayerByName(getLayerId(MockData.BUILDINGS)));
```

Comment:
Sorry, I did not find other mode to really modify the "Queryable" property. Please, how I must do it?